app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TasksController < ApplicationController
+
+  def index
+
+    @tasks = Task.all
+
+  end
+
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+
+    task = Task.new
```

Comment:
To simplify, you can do `Task.new(task_params)` rather than these two separate lines